Library/HUMSlider.m

```diff
@@ -19,14 +21,33 @@
 static CGFloat const HUMImagePadding = 8;
 
 // Sizes
-static CGFloat const HUMTickHeight = 6;
+static CGFloat const HUMTickHeight = 8;
 static CGFloat const HUMTickWidth = 1;
 
+@implementation Tick
+// Constructor for a tick
+- (id)initWithPosition:(double)position {
```

Comment:
a) I would recommend using `CGFloat` rather than `double` for this
b) Would it make more sense to pass in a value along the values included in the slider rather than something between 0 and 1?